corebehrt/main/create_data.py

```diff
@@ -117,22 +118,29 @@ def create_and_save_features(excluder: Excluder, cfg) -> None:
     Returns a list of lists of pids for each batch
     """
     concepts, patients_info = FormattedDataLoader(
-        cfg.paths.data, cfg.loader.concept_types
+        cfg.paths.data,
+        cfg.loader.concept_types,
+        include_values=(
+            cfg.loader.include_values
+            if hasattr(cfg.loader, "include_values") and cfg.loader.include_values
```

Comment:
Can probably just be `getattr(cfg.loader, "include_values", False)` for simplicity

Reply:
Or `cfg.loader.get("include_values")` would work as well?